mypy/checker.py

```diff
@@ -1916,10 +2045,12 @@ def visit_yield_expr(self, e: YieldExpr) -> Type:
     def check_subtype(self, subtype: Type, supertype: Type, context: Context,
                       msg: str = messages.INCOMPATIBLE_TYPES,
                       subtype_label: str = None,
-                      supertype_label: str = None) -> None:
+                      supertype_label: str = None) -> bool:
```

Comment:
I think I've responded to everything. I've only made one small change,
withdraw the VoidType() default in get_generator_return_type(). We can
discuss this separately.